commands/duckpic.js

```diff
@@ -0,0 +1,19 @@
+const axios = require("axios").default
+module.exports = {
```

Comment:
The SlashCommandBuilder import is missing

Reply:
@thieleju oh sorry. i accidentally deleted it while refactoring. thank you for pointing it out